packages/adapters/relayer/src/index.ts

```diff
@@ -72,7 +72,7 @@ export const sendWithRelayerWithBackup = async (
       return { taskId };
     } catch (err: unknown) {
       const jsonError = jsonifyError(err as NxtpError);
-      error_msg = jsonError.message;
+      error_msg = jsonError.context.message ?? jsonError.message;
```

Comment:
May be this should be:
`error_msg = jsonError.context?.message ?? jsonError.message;`